archive-bibliotech.php

```diff
@@ -14,7 +14,7 @@
  *
  * @package WordPress
  * @subpackage Twenty_Twelve
- * @since Twenty Twelve 1.0
+ * @since 1.0
```

Comment:
Is there a separate PR to remove unneeded templates? If so, then great. If not, let's create one. At the end of this, anything that exists should have a proper FileComment block - so it either needs to be deleted, or properly commented.